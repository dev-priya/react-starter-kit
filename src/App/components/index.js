import SimpleComponent from './SimpleComponent';
import StyledComponent from './StyledComponent';

export {
  SimpleComponent,
  StyledComponent
}