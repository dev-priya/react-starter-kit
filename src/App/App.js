import React from 'react';
import {SimpleComponent, StyledComponent} from './components';

// example component
export default () => (
  <div>
    <SimpleComponent message="Simple component demo" />
    <StyledComponent message="Styled component demo" />
  </div>
);