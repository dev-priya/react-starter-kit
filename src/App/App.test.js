import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

// example test file
describe('App', () => {
  const wrapper = shallow(<App />);

  // shallow rendering example
  it('should render a SimpleComponent and a StyledComponent', () => {
    expect(wrapper.find('SimpleComponent').length).toBe(1);
    expect(wrapper.find('StyledComponent').length).toBe(1);
  });

  // snapshot testing example
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});