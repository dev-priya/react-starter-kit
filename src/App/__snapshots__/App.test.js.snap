// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<div>
  <SimpleComponent
    message="Simple component demo"
  />
  <StyledComponent
    message="Styled component demo"
  />
</div>
`;
